test-cases/artifacts/ubuntu2004-fips.yaml

```diff
@@ -0,0 +1,21 @@
+ami_id_db_scylla: 'ubuntu-pro-fips-server/images/hvm-ssd/ubuntu-focal-20.04-amd64-pro-fips-server-20221121-7bc828d1-c072-4d33-a989-fbad50380cfb'  # 'ami-03cf7ddd346310b5f'
+root_disk_size_db: 50
+backtrace_decoding: false
+cluster_backend: 'aws'
+instance_type_db: 'i3.large'
+instance_provision: "spot"
+instance_provision_fallback_on_demand: true
+logs_transport: 'ssh'
+n_db_nodes: 1
+n_loaders: 0
+n_monitor_nodes: 0
+nemesis_class_name: 'NoOpMonkey'
+region_name: 'us-east-1'
+scylla_linux_distro: 'ubuntu-focal'
+scylla_repo: 'http://downloads.scylladb.com/unstable/scylla/master/deb/unified/latest/scylladb-master/scylla.list'
```

Comment:
I think those defaults should go, if someone wants to run it he would need to supply the repo

Reply:
By "those" you mean the repo only, right? Maybe the region, too, but not the rest

Reply:
Removed the scylla_repo